### PR TITLE
refactor, fix, chore(egen): edits get_started_bq_datasets

### PR DESCRIPTION
**Pull request by k-root**
- removes boiler plate and changes according to template
- changes REGION to LOCATION
- changes version for dealing with compatibility error

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/datasets/get_started_bq_datasets.ipynb [hunk 1]
```diff
@@ -74,22 +80,22 @@
...
         "### Objective\n",
         "\n",
-        "In this tutorial, you learn how to use `BigQuery` as a dataset for training with `Vertex AI`.\n",
+        "In this tutorial, you learn how to use BigQuery as a dataset for training with Vertex AI.\n",
         "\n",
         "This tutorial uses the following Google Cloud ML services:\n",
         "\n",
-        "- `Vertex AI Datasets`\n",
-        "- `BigQuery Datasets`\n",
+        "- Vertex AI Datasets\n",
```
Comment (kittyabs) [inline notebooks/official/datasets/get_started_bq_datasets.ipynb:87]: - change: "Vertex AI Datasets"
-  to: "Vertex AI datasets"

File: notebooks/official/datasets/get_started_bq_datasets.ipynb [hunk 2]
```diff
@@ -74,22 +80,22 @@
...
+        "- BigQuery Datasets\n",
```
Comment (kittyabs) [inline notebooks/official/datasets/get_started_bq_datasets.ipynb:88]: - change: "BigQuery Datasets"
- to:  "BigQuery datasets"

File: notebooks/official/datasets/get_started_bq_datasets.ipynb [hunk 3]
```diff
@@ -74,22 +80,22 @@
...
         "The steps performed include:\n",
         "\n",
-        "- Create a Vertex AI `Dataset` resource from `BigQuery` table -- compatible for `AutoML` training.\n",
-        "- Extract a copy of the dataset from `BigQuery` to a CSV file in Cloud Storage -- compatible for `AutoML` or custom training.\n",
-        "- Select rows from a `BigQuery` dataset into a `pandas` dataframe -- compatible for custom training.\n",
-        "- Select rows from a `BigQuery` dataset into a `tf.data.Dataset` -- compatible for custom training `TensorFlow` models.\n",
-        "- Select rows from extracted CSV files into a `tf.data.Dataset` -- compatible for custom training `TensorFlow` models.\n",
-        "- Create a `BigQuery` dataset from CSV files.\n",
-        "- Extract data from `BigQuery` table into a `DMatrix` -- compatible for custom training `XGBoost` models."
+        "- Create a Vertex AI Dataset resource from BigQuery table -- compatible for AutoML training.\n",
```
Comment (kittyabs) [inline notebooks/official/datasets/get_started_bq_datasets.ipynb:92]: - change: "Vertex AI Datasets"
- to: "Vertex AI datasets"

File: notebooks/official/datasets/get_started_bq_datasets.ipynb [hunk 4]
```diff
@@ -103,7 +109,7 @@
         "When doing E2E MLOps on Google Cloud, following are the best practices when dealing with structured (tabular) data in BigQuery:\n",
         "\n",
         "- For AutoML training:\n",
-        "  - Create a managed dataset with Vertex AI `TabularDataset`.\n",
+        "  - Create a managed dataset with Vertex AI TabularDataset.\n",
```
Comment (kittyabs) [inline notebooks/official/datasets/get_started_bq_datasets.ipynb:112]: - change: "Vertex AI TabularDataset"
- to: Vertex AI tabular dataset"

For reference, see: https://cloud.google.com/vertex-ai/docs/tutorials/tabular-bq-prediction/create-dataset

File: notebooks/official/datasets/get_started_bq_datasets.ipynb [hunk 5]
```diff
@@ -466,10 +418,10 @@
         "\n",
         "#### BigQuery input data\n",
         "\n",
-        "Next, create the `Dataset` resource using the `create` method for the `TabularDataset` class, which takes the following parameters:\n",
+        "Next, create the Dataset resource using the `create` method for the `TabularDataset` class, which takes the following parameters:\n",
```
Comment (kittyabs) [inline notebooks/official/datasets/get_started_bq_datasets.ipynb:421]: - change: "Dataset resource"
- to: "dataset resource"

File: notebooks/official/datasets/get_started_bq_datasets.ipynb [hunk 6]
```diff
@@ -466,10 +418,10 @@
...
         "\n",
-        "- `display_name`: The human readable name for the `Dataset` resource.\n",
-        "- `bq_source`: Import data items from a BigQuery table into the `Dataset` resource.\n",
+        "- `display_name`: The human readable name for the Dataset resource.\n",
```
Comment (kittyabs) [inline notebooks/official/datasets/get_started_bq_datasets.ipynb:423]: - change: "Dataset resource"
- to: "dataset resource"

File: notebooks/official/datasets/get_started_bq_datasets.ipynb [hunk 7]
```diff
@@ -466,10 +418,10 @@
...
+        "- `bq_source`: Import data items from a BigQuery table into the Dataset resource.\n",
```
Comment (kittyabs) [inline notebooks/official/datasets/get_started_bq_datasets.ipynb:424]: - change: "Dataset resource"
- to: "dataset resource"

File: notebooks/official/datasets/get_started_bq_datasets.ipynb [hunk 8]
```diff
@@ -539,10 +491,10 @@
         "\n",
         "#### CSV input data\n",
         "\n",
-        "Next, create the `Dataset` resource using the `create` method for the `TabularDataset` class, which takes the following parameters:\n",
+        "Next, create the Dataset resource using the `create` method for the `TabularDataset` class, which takes the following parameters:\n",
```
Comment (kittyabs) [inline notebooks/official/datasets/get_started_bq_datasets.ipynb:494]: - change: "Dataset resource"
- to: "dataset resource"

File: notebooks/official/datasets/get_started_bq_datasets.ipynb [hunk 9]
```diff
@@ -539,10 +491,10 @@
...
         "\n",
-        "- `display_name`: The human readable name for the `Dataset` resource.\n",
-        "- `gcs_source`: A list of one or more dataset index files to import the data items into the `Dataset` resource.\n",
+        "- `display_name`: The human readable name for the Dataset resource.\n",
```
Comment (kittyabs) [inline notebooks/official/datasets/get_started_bq_datasets.ipynb:496]: - change: "Dataset resource"
- to: "dataset resource"

File: notebooks/official/datasets/get_started_bq_datasets.ipynb [hunk 10]
```diff
@@ -539,10 +491,10 @@
...
+        "- `gcs_source`: A list of one or more dataset index files to import the data items into the Dataset resource.\n",
```
Comment (kittyabs) [inline notebooks/official/datasets/get_started_bq_datasets.ipynb:497]: - change: "Dataset resource"
- to: "dataset resource"

**Review by kittyabs — COMMENTED** [review]
numerous required edits

File: notebooks/official/datasets/get_started_bq_datasets.ipynb (hunk 4 above)
Comment (k-root) [reply notebooks/official/datasets/get_started_bq_datasets.ipynb:112]: Question: Here `TabularDataset` is a python class. Shouls that still be changed?

**Review by katiemn — APPROVED** [review]

**Pull request merged**